recipes/blast/meta.yaml

```diff
@@ -25,7 +25,8 @@ source:
       - normbase.patch
 
 build:
-  number: 1
+  skip: True # [osx]
```

Comment:
Would be nice next time to add a link to the PR/issue describing why it fails :)